### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tanushree57**
Added Suzlon under Energy category

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +130,14 @@ Short description.
 - [Vestas](https://www.vestas.com/) | Denmark | 1945
 
   Manufacturer, seller, installer, and servicer of wind turbines. As of 2019 Vestas has installed over 66,000 wind turbines for a capacity of 100 GW in over 80 countries on five continents.
+  
+- [Suzlon Energy Ltd.](https://www.suzlon.com/) | India | 1995
+
+Suzlon is a win turbine manufacturer. It is present in 18 countries across the continents.
```
Comment (nglgzz) [inline README.md]:
```suggestion
Wind turbine manufacturer present in 18 countries across the continents.
```

File: README.md [hunk 2]
```diff
@@ -130,6 +130,14 @@ Short description.
...
+
+- [Siemens](https://new.siemens.com/in/en.html) | Germany | 1847
+
+It manufactures wind turbines and provides onshore and offshore wind services. It is the world's second largest wind turbine manufacturer.
```
Comment (nglgzz) [inline README.md]: According to https://www.statista.com/statistics/271031/market-share-of-the-largest-wind-turbine-manufacturers-in-the-world/

```suggestion
Multinational manufacturing company. Among their many projects they produce wind turbines and provide onshore and offshore wind services. It is the world's third largest wind turbine manufacturer as of 2018.
```

**Review by nglgzz — CHANGES_REQUESTED** [review]
Could you move your entries so they are in alphabetical order? Also I suggested a couple of other changes.

**Comment by tanushree57** [comment]
@nglgzz I've sorted them alphabetically and accepted the necessary changes

**Review by nglgzz — APPROVED** [review]

**Comment by nglgzz** [comment]
Thank you for your contribution @tanushree57 :sparkles:

**Pull request merged**